src/vehicleFc.cpp

```diff
@@ -0,0 +1,52 @@
+#include "vehicle.h"
+
+#if (CURRENT_VEHICLE == FC)
+
+// sensor definitions
+SensorGps gps(new SFE_UBLOX_GNSS(), GPS_UPDATE_FREQUENCY);
+SensorThermo thermo1(&SPI, A5);
+SensorThermo thermo2(&SPI, A4);
+SensorSigStrength sigStrength;
+SensorVoltage inVoltage;
+
+// command definitions
+SensorCommand<SensorGps, String> gpsLat(&dataQ, &gps, "lat", &SensorGps::getLatitude, 1);
+SensorCommand<SensorGps, String> gpsLong(&dataQ, &gps, "long", &SensorGps::getLongitude, 1);
+SensorCommand<SensorGps, String> gpsVertAccel(&dataQ, &gps, "v-accel", &SensorGps::getVerticalAcceleration, 2);
+SensorCommand<SensorGps, String> gpsHorAccel(&dataQ, &gps, "h-accel", &SensorGps::getHorizontalAcceleration, 2);
+SensorCommand<SensorThermo, String> thermoTemp1(&dataQ, &thermo1, "temp1", &SensorThermo::getProbeTemp, 5);
+SensorCommand<SensorThermo, String> thermoTemp2(&dataQ, &thermo2, "temp2", &SensorThermo::getProbeTemp, 5);
+
+// array definitions: update counts if you make changes here
+Sensor *sensors[] = {&gps, &thermo1, &thermo2, &sigStrength, &inVoltage};
+IntervalCommand *commands[] = { &gpsLat, &gpsLong, &gpsVertAccel, &gpsHorAccel, &thermoTemp1, &thermoTemp2 };
+uint16_t sensor_count = 5;
+uint16_t command_count = 6;
```

Comment:
Can we put these as macros or in the array type itself?

Reply:
I don't see a way to do that without putting the sensor count in a different file. Do you have any suggestions for how to do this cleanly?

Reply:
Created issue #61 for this

Reply:
> I don't see a way to do that without putting the sensor count in a different file. Do you have any suggestions for how to do this cleanly?

There was some residual discussion about this in #50, but I think using C++ STL containers would be the way to go (so either an `array` or `vector`).